src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.cs

```diff
@@ -119,12 +126,15 @@ public override SyntaxNode GetPropertyDeclaration(SyntaxToken token)
             IFieldSymbol propertyBackingField,
             IMethodSymbol setMethod,
             string desiredSetMethodName,
+            bool copyLeadingTrivia,
             CancellationToken cancellationToken)
         {
             var methodDeclaration = GetSetMethodWorker(
                 generator, propertyDeclaration, propertyBackingField,
                 setMethod, desiredSetMethodName, cancellationToken);
 
+            methodDeclaration = CopyLeadingTrivia(propertyDeclaration, methodDeclaration, copyLeadingTrivia);
```

Comment:
Seems weird to pass a boolean into a method telling the method whether it should do anything. Why not check the boolean here? I had to go looking to see if the call had side effects when copyLeadingTrivia is false, and it would have been easier if it just wasn't called in that case. `methodDeclaration = copyLeadingTrivia ? CopyLeadingTrivia(propertyDeclaration, methodDeclaration) : methodDeclaration`?